src/Processor/ProcessorInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Webhook Module for PHP
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Webhook\Processor;
+
+interface ProcessorInterface
+{
+    public function process(): void;
```

Comment:
Should we have this function return something so that the integration can update the order according to the result? Or is that going to be done in a different way?

Reply:
[Current implementation](https://github.com/Adyen/adyen-shopware6/pull/190/files#diff-d72bb3f47aa9e305e5a7b625f9e491fa1c78771bdce76ca9607f7dab67db6154R153-R154) is to set the transition state in `process()` and then call the getter. But now that I take another look at it, it does make sense to simply return the new state from the `process()` function